src/test/java/api/support/matchers/EventMatchers.java

```diff
@@ -0,0 +1,125 @@
+package api.support.matchers;
+
+import static com.jayway.jsonpath.matchers.JsonPathMatchers.hasJsonPath;
+import static org.folio.circulation.support.JsonPropertyFetcher.getBooleanProperty;
+import static org.hamcrest.core.Is.is;
+
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeDiagnosingMatcher;
+
+import io.vertx.core.json.JsonObject;
+
+public class EventMatchers {
+
+  public static Matcher<JsonObject> isValidItemCheckedOutEvent(JsonObject loan) {
+    return JsonObjectMatcher.allOfPaths(
+      hasJsonPath("eventType", is("ITEM_CHECKED_OUT")),
+      hasJsonPath("eventPayload", isValidItemCheckedOutEventPayload(loan))
```

Comment:
it can be refactored to something like:
```suggestion
      hasJsonPath("eventPayload", allOf(
         hasJsonPath("userId", is(loan.getString("userId"))),
         hasJsonPath("loanId", is(loan.getString("id"))),
         ...
      ))
```

Reply:
Refactored